cmd/start.go

```diff
@@ -44,13 +46,17 @@ func init() {
 	startCmd.Flags().String("crd-version", "v1", "the version of the CRD you want monitored")
 	startCmd.Flags().String("crd-namespace", metav1.NamespaceNone, "(optional) the namespace of the CRD you want monitored, only needed for namespaced CRDs (ex: default)")
 	startCmd.Flags().String("templates", "", "absolute path to the directory with your template files")
+	startCmd.Flags().String("metrics-address", ":8080", "The address and port for the metrics endpoint")
```

Comment:
Is the address required? If one isn't set does it default to 0.0.0.0? maybe our default should be to listen everywhere?

Reply:
The address is not required.  Its an IP and port combination, if only a port is passed in it still listens on 0.0.0.0 and defaults to port 80 with no arguments.